test/inductor/test_flex_decoding.py

```diff
@@ -1573,6 +1585,23 @@ def test_mixed_dtypes_fails(self, device):
         ):
             flex_attention(query, key, value, _identity)
 
+    @supported_platform
+    @skipCUDAIf(True, "Not supported on CUDA")
```

Comment:
Why it's not supported by any of accelerators?

Reply:
This is because that the template parameter `partition_size` is only used for CPU, which is created during lowering. https://github.com/pytorch/pytorch/blob/868cf83793c4584cec67baa7906959c127a38269/torch/_inductor/kernel/flex/flex_cpu.py#L293